android/src/main/java/com/onegini/mobile/sdk/reactnative/RNOneginiSdk.kt

```diff
@@ -141,7 +142,7 @@ class RNOneginiSdk(reactContext: ReactApplicationContext) : ReactContextBaseJava
     override fun isFingerprintAuthenticatorRegistered(profileId: String, promise: Promise) {
         try {
             promise.resolve(authenticatorManager.isFingerprintAuthenticatorRegistered(profileId))
-        } catch (e: OneginiError) {
+        } catch (e: OneginReactNativeException) {
```

Comment:
is this the only exception that can be thrown here?

Reply:
yes